config/schema/metatag.metatag_tag.schema.yml

```diff
@@ -0,0 +1,19 @@
+metatag.metatag_tag.*:
```

Comment:
If I understand the schema files correctly, this is the plugin configuration. As defined by `ConfigurablePluginInterface` this should be a non-list associative array (a.k.a. hash map).

I can also recommend following the naming scheme as documented in http://cgit.drupalcode.org/plugin/tree/PLUGIN_TYPES.md. I'm hoping to make this part of Drupal 8.1.0 as well in order to improve generic plugin handling.